cleanup.js

```diff
@@ -1,6 +1,17 @@
 const core = require('@actions/core');
 const { shell } = require('./shell.js');
 
+if (core.getState("EG_FAILED") == "true") {
+    if (core.getInput('submit_diagnostics_on_failure') == "true") {
+        try {
+            shell('egctl advanced log-upload');
+        } catch (error) { }
+    }
+    try {
+        shell('cat /var/log/edgeguardian/datapath.log || true;')
```

Comment:
This may be considered too revealing, we may need to do some sanitation.

Reply:
Good point, I am erring on the side of debugging now.  I can relax this once we debug the current issue.